cif_pow.dic

```diff
@@ -6291,6 +6294,27 @@ save_PD_DIFFRACTOGRAM
 
 save_
 
+save_pd_diffractogram.diffrn_frame_data_id
+
```

Comment:
As I commented elsewhere, I think this definition is unnecessary.

Reply:
removed